sdk/logical/managed_key.go

```diff
@@ -16,6 +16,9 @@ const (
 	KeyUsageVerify
 	KeyUsageWrap
 	KeyUsageUnwrap
+	KeyUsageMACGenerate
+	KeyUsageMACVerify
```

Comment:
Should these just be sign/verify?

Reply:
Yeah, we could just simplify it. I only added these because some KMS solutions have more granular permissions. I don't have a strong opinion either way.